modules/client-configuration/pages/supported-features.adoc

```diff
@@ -152,6 +152,13 @@ ifeval::[{uyuni-content} == true]
 | {check}
 endif::[]
 
+ifeval::[{uyuni-content} == true]
+| {alibabaclo} 2
+| {x86_64}
+| {check}
```

Comment:
Shouldn't this (traditional) be `cross`? I think we have a bug at Amazon Linux as well. Feel free to fix it :-)

Reply:
Do not fix Amazon Lunux, I already did it at https://github.com/uyuni-project/uyuni-docs/pull/883

Fixing Alibaba to have `{cross}` for traditional is enough.

Reply:
Changed Alibaba